test/README.md

```diff
@@ -1,5 +1,63 @@
 ## End-to-End Testing
 
+### AWS E2E Testing
+Featured in this directory is a `./awsE2ESolutionSetup` script whose goal is to encapsulate deploying an E2E environment in AWS. This script will perform the following key actions:
+1. Deploy an Elasticsearch source cluster to EC2 using the following [CDK](https://github.com/lewijacn/opensearch-cluster-cdk/tree/migration-es)
```

Comment:
Looks like you liked your personal fork; was this intentional?

Reply:
Yes this is where we have the ES EC2 CDK currently

Reply:
I'll be curious to hear why this is the case, but given this is a documentation task it's not a blocker for me.